include/sdf/Param.hh

```diff
@@ -518,13 +766,15 @@ namespace sdf
       std::string typeStr = this->dataPtr->TypeToString<T>();
       if (typeStr.empty())
       {
-        sdferr << "Unknown parameter type[" << typeid(T).name() << "]\n";
+        _errors.push_back({ErrorCode::PARAMETER_ERROR,
```

Comment:
Would a new error code, `UNKNOWN_PARAMETER_TYPE` make more sense?

Reply:
Definitely, I  added the new error type and I also used it here: https://github.com/ignitionrobotics/sdformat/blob/a7ee2929e5e51bb578ef9a0f85ad18df76d9fc1d/src/Param.cc#L995